README.md

```diff
@@ -133,6 +137,11 @@ JWT_SECRET=EverythingisAwesome
 PORT=8000
 ```
 
+If you would like to help *contribute* to this module,
```

Comment:
Nice. We should add this as a default to the bottom of every repo. Another mini script to write - the `dwyl readme initialiser` :wink: